core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -37,7 +37,12 @@
      * Creates a new factory that creates a non-daemon and normal-priority thread.
      *
      * @param threadNamePrefix the prefix of the names of the threads created by this factory.
+     *
+     * @deprecated Use {@link ThreadFactories#newEventLoopThreadFactory(String, boolean)} or
```

Comment:
`(String, boolean)` -> `(String)`

Reply:
Question : We don't have a shortcut method `newEventLoopThreadFactory(String)`.
                 Then, should I remove javadoc phrase or add a new method?

Reply:
I misread the code, sorry about that. 😄 @ikhoon will leave a comment for this. 😉

Reply:
Thank you for review, and quick answer! :+1:

Reply:
`newEventLoopThreadFactory` does not have an overload method for `(String)`.
I think this documentation can confuse a user who creates `EventLoopThreadFactory` with only  `threadNamePrefix`.

Could you update your documentation to `{@linkplain ThreadFactories#newEventLoopThreadFactory(String, boolean) ThreadFactories#newEventLoopThreadFactory(String, false)}` for guiding a user who uses default value.

Reply:
Updated it. :smile: